### DISCUSSION TITLE
Can last versions of keycloak work with Ms SQL Server 2019?

### DISCUSSION THREAD

**Discussion by JordiAlonsoF** [keycloak/keycloak#17240] [Archive]
My company wants to implement keycloack. I'd want to know if last versions of keycloak can use a Ms SQL Server 2019 instance.
According to this link "https://www.keycloak.org/server/db" only SQL 2016 has been tested.

**Comment by vramik** [comment]
Yes, it is currently tested with mssql2019. Please see: https://github.com/keycloak/keycloak/blob/8abe984844c53f55294eedc18939f2b603a3a9d7/pom.xml#L157-L158

https://www.keycloak.org/server/db seems to outdated.
